devices/debian_wifi.py

```diff
@@ -1,88 +1,126 @@
-import re
+import re, time
 import debian
 import pexpect
+from countrycode import countrycode
+from lib.wifi import wifi_client_stub
+import pdb
 
-class DebianWifi(debian.DebianBox):
+class DebianWifi(debian.DebianBox, wifi_client_stub):
     '''Extension of Debian class with wifi functions'''
 
     model = ('debianwifi')
+    wifi_interface = "wlan0"
+    wifi_interface1 = "wlan1"
 
-    iface_wlan = "wlan1"
+    wifi_client_control = None
 
-    def scan(self):
+    def disable_and_enable_wifi(self, iface):
```

Comment:
On the stub commit I suggest breaking these into enable/disable functions not combining both. They will be generically more useful.

Reply:
Ack